MerlinAU.sh

```diff
@@ -5322,15 +5340,18 @@ _GetOfflineFirmwareVersion_()
             printf "\nFailed to identify firmware version from the ZIP file name."
         fi
         printf "\nPlease enter the firmware version number in the format ${fwVersionFormat}\n"
-        printf "\n(Examples: 3004.388.8.0 or 3004.388.8.0_beta1):  "
+        printf "\n(Examples: 3004.388.8.0 or 3004.388.8.0_beta1). Enter 'e' to exit:  "
         read -r formatted_version
 
         # Validate user input #
         while ! echo "$formatted_version" | grep -qE "^${validate_version_regex}$"
         do
+            if echo "$formatted_version" | grep -qE "^(e|E|exit|Exit)$"; then
+                return 1
+            fi
```

Comment:
Allow the user to exit this screen without making a selection. (Return to advanced menu screen)

Reply:
> Allow the user to exit this screen without making a selection. (Return to advanced menu screen)
>
Good point.